trieste/acquisition/models/gpflux/sampler.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 The Trieste Contributors
```

Comment:
sorry, what I meant was `trieste/models/gpflux/` not as a subfolder in `acquisition` folder